### ISSUE TITLE
Is there a way to do it with docker compose?

### ISSUE THREAD

**Issue by adi-kmt** [usmanzaheer1995/spring-boot-demo#1]
Can I use?
`docker compose up`

**Comment by usmanzaheer1995** [comment]
@adi-kmt no, unfortunately. The `docker compose up` command will only run a postgresql instance locally which you can use to then start the app.

You can also just do `make start/db` which will docker compose up.

**Comment by adi-kmt** [comment]
@usmanzaheer1995 I'm getting an error at 
`RUN apk --no-cache add make` in dockerfile-local
with error `ERROR: failed to solve: process "/bin/sh -c apk --no-cache add make" did not complete successfully: exit code: 127`

**Comment by adi-kmt** [comment]
@usmanzaheer1995 
`make build/image/local` seems to have got stuck at 
I'm getting an error `COPY --from=builder /app/build/libs/spring-boot-demo-local.jar /app/spring-boot-demo-local.jar`, getting failed to compute cache key

**Comment by usmanzaheer1995** [comment]
@adi-kmt To run it locally, you can follow the steps in the Readme (making sure that you run `make start/db`.

It is strange that you get an error while trying to build the local image, as I just tried running it using `make build/image/local` on my ubuntu machine just now and it works fine.

But running it on my mac machine, i get an error while gradle is building. I will look into it.

**Comment by adi-kmt** [comment]
@usmanzaheer1995 got it to work on ubuntu [with jdk 17](https://github.com/adi-kmt/sample-jooq-kotlin-springboot-docker-flyway)
I will probably try to implement a seperate docker for postgres and push

**Comment by usmanzaheer1995** [comment]
There was an issue with the alpine image not working on mac, I have finally found a fix and pushed it so should work on all platforms now.
